### PR TITLE
Check value is a number.

### PR DESCRIPTION
**Pull request by michalstrzelecki**

### CODE REVIEW

File: src/Gzero/Core/Parsers/NumericParser.php [hunk 1]
```diff
@@ -112,6 +114,9 @@ public function parse(Request $request)
             } else {
                 $this->value = $value;
             }
+
+            // Need it becouse of Cyclomatic Complexity.
```
Comment (PSkierniewski) [inline src/Gzero/Core/Parsers/NumericParser.php]: Typo, it should be `because` instead

File: src/Gzero/Core/Parsers/NumericParser.php [hunk 2]
```diff
@@ -95,8 +97,8 @@ public function parse(Request $request)
 
             // do not reorder this
             if (substr($value, 0, 1) === '!') {
-                $this->operation = '!=';
-                $this->value     = substr($value, 1);
+                $this->operation = '!';
```
Comment (PSkierniewski) [inline src/Gzero/Core/Parsers/NumericParser.php]: Why you change operation, now it's not matching $allowedOperations in Query\Condition - this should be a valid database operator

File: tests/unit/NumericParserTest.php [hunk 3]
```diff
@@ -70,6 +70,19 @@ public function itCanParseLessThan()
         $this->assertEquals(123, $parser->getValue());
     }
 
+    /** @test */
+    public function itCanNotParseStringValue()
```
Comment (PSkierniewski) [inline tests/unit/NumericParserTest.php]: We need at least one test from using API, e.g for file size in Files tests to make sure that it works with URL parameters.

Comment (michalstrzelecki) [reply tests/unit/NumericParserTest.php]: It is in commit below.

**Pull request merged**